app/controllers/application_controller.rb

```diff
@@ -19,7 +20,34 @@ def get_all_facilities
       #This can be written as follows:
       # session[:login_status] = 'not logged in' unless session[:login_status] == 'logged in'
       # @allfacilities = Facility.all
-
+    
   	  
   end
+  def get_provided_services
```

Comment:
This should be defined in the model, so that the controller just calls it. 

Further, you can extract this method into a FacilitiesServicesController under the index method.

Remember, Never Write Custom Actions? lol